src/routes/app/+page.ts

```diff
@@ -1,11 +1,30 @@
-import { search } from "$lib/api/documents.js";
+import type { SearchOptions } from "$lib/api/types";
+import { search } from "$lib/api/documents";
 
 export async function load({ url, fetch }) {
   const query = url.searchParams.get("q") || "";
-  const searchResults = search(query, true, fetch);
+  const per_page = +url.searchParams.get("per_page") || 25;
```

Comment:
I'm unfamiliar with the `+url` syntax, what's that about?

Reply:
It's just turning a string into a number